apps/site/assets/js/test/psl-page-setup_test.js

```diff
@@ -39,7 +39,7 @@ const pslHtml = `
   </div>
 `;
 
-describe.only("psl-filters", () => {
+describe("psl-filters", () => {
```

Comment:
```suggestion
describe.only("psl-filters", () => {
```
There are a couple tests to fix as a result of this, so should be a separate PR